Fix up docs for 2026.1 (#19890)
nvaccess/nvda: user_docs/en/userGuide.md

Reviewer: Could this line be clarified? The "in NVDA" here, reads strangely to me. Presumably it's the "node in NVDA", is it actually a virtualized node in NVDA, as the user understands it?
Would it be better to say "the current formula node" or some such, and leave off the "in NVDA"?
Not leaving a suggestion, because this sounds odd, but I'm not sure what's better. I'm not someone who works with formal math a lot.

Reviewer 2: Yes, the concept of "node" has not been introduced anywhere previously and is not something meant to be known by end users.

Below is a summary of the most useful commands for those wanting to quickly get started with navigating mathematical content.

> * Use the arrow keys to move left/right or up/down a mathematical structure (e.g. into/out of a fraction).

Reviewer: Should this say "in" a structure?

```suggestion
* Use the arrow keys to move left, right, up or down in a mathematical structure (e.g. into/out of a fraction).
```

> * If inside a table, `control+arrow` keys will move by cell.
> * `home` / `end` moves to the start/end of the expression.
> * `space` reads your current position.
> * `shift+upArrow` / `shift+downArrow` will change the mode of navigation, which will be discussed in more detail in the following section.

Reviewer: I'm not sure if we're trying to keep this section less formal for a reason. But if not:

```suggestion
* If inside a table, `Control+arrow` keys will move by cell.
* `home` or `end` keys will move to the start or end of an expression, respectively.
* `space` reads your current position.
* `shift+upArrow` and `shift+downArrow` will change the mode of navigation, which will be discussed in the next section.
```

I have left the keys lower cased, contrary to the Microsoft standard, since that's common in the UG.
Removed the "in more detail" from the last item, because it hasn't been discussed in any detail so far.
